tests/accessibility/.pa11y-ci.js

```diff
@@ -32,19 +32,23 @@ module.exports = {
     performCheck(chooseATemplatePage(baseUrl)),
     performCheck(chooseATemplatePageError(baseUrl)),
 
+    // NHS App journey
     performCheck(createNHSAppTemplatePage(baseUrl)),
     performCheck(createNHSAppTemplateErrorPage(baseUrl)),
     performCheck(reviewNHSAppTemplatePage(baseUrl)),
     performCheck(reviewNHSAppTemplateErrorPage(baseUrl)),
     performCheck(submitNHSAppTemplatePage(baseUrl)),
-    performCheck(NHSAppTemplateSubmittedPage(baseUrl)),
+    performCheck(NhsAppTemplateSubmittedPage(baseUrl)),
```

Comment:
rename to `nhsAppTemplateSubmittedPage` to be consistent with the other submitted pages. Although I think we should rename all the submitted pages to
`submitted{TemplateType}TemplatePage` - all the other pages kind of follow this pattern.

Reply:
The name are consistent (unless I'm missing something). I think this was just the capitalization on the function name. The name of the functions match the name of the URLs they relate to (and the folder name names in the app). 

I.E.

- `nhsAppTemplateSubmittedPage` -> `/templates/nhs-app-template-submitted`
- `textMessageTemplateSubmittedPage` -> `/templates/text-message-template-submitted`
- `emailTemplateSubmittedPage` -> `/templates/email-template-submitted`

Reply:
all the other pages in the journey follow this pattern
{pageType}{templateType}Page 
so create, review and submit all do this. Only the submitted page goes the other way. But if they match the URLs then that's better off. Didn't look at that.